### PR TITLE
Fix more 404 links and a few redirects.

### LINKED ISSUE #2603: Web page broken link (404 error).
Web page:
[https://nodejs.org/en/knowledge/advanced/streams/what-are-streams/](https://nodejs.org/en/knowledge/advanced/streams/what-are-streams/)

The above page contains the below link that results in 404 page.
[pipe](https://nodejs.org/how-to-use-stream-pipe)

### LINKED ISSUE #2613: Web page broken link (404 error) ../what-is-try-catch/
Web page:
[https://nodejs.org/en/knowledge/errors/what-is-try-catch/](https://nodejs.org/en/knowledge/errors/what-is-try-catch/)

The above page contains the below link that results in 404 page.
[standard](https://nodejs.org/what-are-the-error-conventions)

### PR DESCRIPTION
**Pull request by XhmikosR**
Fixes #2603, fixes #2613 along the way.

### CODE REVIEW

**Comment by XhmikosR** [comment]
I also have https://github.com/nodejs/nodejs.org/issues/2424#issuecomment-533997529 that I need guidance how to proceed. And those are the English ones only, I suspect there are even more.

File: locale/en/knowledge/getting-started/globals-in-node-js.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ Node.js has a number of built-in global identifiers that every Node.js developer
...
+- `process` - Node's built-in `process` module, which provides interaction with the current Node process.  [Read More](/en/knowledge/getting-started/the-process-module/)
+- `console` - Node's built-in `console` module, which wraps various STDIO functionality in a browser-like way.  [Read More](/en/knowledge/getting-started/the-console-module/)
+- `setTimeout()`, `clearTimeout()`, `setInterval()`, `clearInterval()` - The built-in timer functions are globals. [Read More](/en/knowledge/javascript-conventions/what-are-the-built-in-timer-functions/)
 
 As mentioned above, there are also a number of 'pseudo-globals' included at the module level in every module:
 
-- `module`, `module.exports`, `exports` - These objects all pertain to Node's module system.  [Read More](/en/knowledge/getting-started/what-is-require)
-- __filename - The `__filename` keyword contains the path of the currently executing file.  Note that this is not defined while running the [Node REPL](/en/knowledge/REPL/how-to-use-nodejs-repl).
```
Comment (XhmikosR) [inline locale/en/knowledge/getting-started/globals-in-node-js.md:23]: Fixed the wrong output in these 2 lines too

**Comment by Trott** [comment]
@nodejs/website This could use reviews.

**Review by alexandrtovmach — APPROVED** [review]
Thanks a lot for your effort

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Pull request merged**